docs/software/system_stats.md

```diff
@@ -251,4 +252,52 @@ Official website: <https://webmin.com/>
 Official documentation: <https://doxfer.webmin.com/Webmin/Main_Page>  
 Wikipedia: <https://wikipedia.org/wiki/Webmin>
 
+
+## K3s
+
+Lightweight Kubernetes - The certified Kubernetes distribution built for IoT & Edge computing
+
+![K3s Logo](https://k3s.io/images/logo-k3s.svg){: width="300" height="116" loading="lazy"}
```

Comment:
Let's store the logo on our repo, so we do not depend on K3s server being up. I checked the trademark rules and generally the Linux Foundation allows to use their logos (in original aspect ratio without visual changes) to link to their homepage, so should be fine: https://www.linuxfoundation.org/trademark-usage

Reply:
Makes sense, I'll add the logo under `assets`.